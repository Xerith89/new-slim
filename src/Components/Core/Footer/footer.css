footer {
    bottom: 0;
    height: 60px;
    width: 100%;
    position: absolute;
    color: #ffffff;
}

