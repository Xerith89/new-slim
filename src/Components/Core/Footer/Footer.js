import React from 'react'
import './footer.css';
import logo from '../../../images/footer_logo.png';

export default function Footer() {
    return (
        <div>
            <footer data-testid="footer">
                <nav className="navbar ml-auto navbar-dark bg-dark ">
                    <span><img src={logo} alt="footerlogo"/></span>
                    <span>
                        <div>
                            <small className="ml-5">Registered office: 145 New Dover Road, Capel-Le-Ferne, Folkestone CT18 7JR</small> <br/>
                            <small className="ml-5">Stephen Lower Insurance Services Limited is authorised and regulated by the Financial Conduct Authority, Financial Services Register number 628613
                            Registered in England and Wales No.4930449.</small><br/>         
                        </div>
                    </span>
                </nav>
            </footer>
        </div>
    )
}
