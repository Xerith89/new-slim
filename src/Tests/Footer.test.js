import React from 'react';
import Footer from '../Components/Core/Footer/Footer';
import {render, cleanup, fireEvent} from '@testing-library/react';


afterEach(cleanup)

test('Footer component renders to the DOM', () => {
    const {getByTestId} = render(<Footer />);
    expect(getByTestId('footer')).toMatchSnapshot();
});