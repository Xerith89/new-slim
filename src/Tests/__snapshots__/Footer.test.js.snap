// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders to the DOM 1`] = `
<footer
  data-testid="footer"
>
  <nav
    class="navbar ml-auto navbar-dark bg-dark "
  >
    <span>
      <img
        alt="footerlogo"
        src="footer_logo.png"
      />
    </span>
    <span>
      <div>
        <small
          class="ml-5"
        >
          Registered office: 145 New Dover Road, Capel-Le-Ferne, Folkestone CT18 7JR
        </small>
         
        <br />
        <small
          class="ml-5"
        >
          Stephen Lower Insurance Services Limited is authorised and regulated by the Financial Conduct Authority, Financial Services Register number 628613 Registered in England and Wales No.4930449.
        </small>
        <br />
      </div>
    </span>
  </nav>
</footer>
`;
